fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/util/HashUtils.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * HashUtils.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2020 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.util;
+
+import com.apple.foundationdb.record.QueryHashable;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+
+/**
+ * Utility class for hash functions.
+ */
+public class HashUtils {
+
+    private HashUtils() {}
```

Comment:
Doesn't that give you a warning? Like IDEA wants you to comment on why this is empty.

Reply:
No, no warnings. @MMcM asked for a private constructor for all-static-methods class.
Is there a particular annotation or something I need to add?